platform/src/main/java/org/stellar/anchor/platform/component/share/ClientsBeans.java

```diff
@@ -0,0 +1,15 @@
+package org.stellar.anchor.platform.component.share;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.stellar.anchor.platform.config.ClientsConfig;
+
+@Configuration
+public class ClientsBeans {
+  @Bean
+  @ConfigurationProperties(prefix = "")
```

Comment:
Is this intentional? The default value is already `""`.

Reply:
Sprint gives contradicting warnings. Removing the `""` gives another warning.